packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -228,4 +238,28 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
             this.selectWidgetFileNode(this.shell.currentWidget);
         }
     }
+
+    /**
+     * Collapse navigator nodes and set focus on first visible node
+     * - single root workspace: collapse all nodes except root
+     * - multiple root workspace: collapse all nodes, even roots
+     */
+    async collapseAll(): Promise<void> {
```

Comment:
`collapseFileTree`

Reply:
are you fine with the updated change `collapseNavigatorTree` since it is the `navigator-widget` and `@theia/navigator` extension?